integration_test/nodes/init.sh

```diff
@@ -8,7 +8,7 @@ DIR=$(
 )
 
 # Download solc for PlonkVerifier
-$TAIKO_MONO_DIR/packages/protocol/script/download_solc.sh
+#$TAIKO_MONO_DIR/packages/protocol/script/download_solc.sh
```

Comment:
Let's just delete it?

Reply:
Has given a check in entrypoint.sh about solc tool.